app/src/main/java/com/sopt/kakaobank/presentation/history/HistoryFragment.kt

```diff
@@ -1,11 +1,113 @@
 package com.sopt.kakaobank.presentation.history
 
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
 import com.sopt.kakaobank.R
 import com.sopt.kakaobank.core.base.BindingFragment
+import com.sopt.kakaobank.core.util.fragment.statusBarColorOf
+import com.sopt.kakaobank.data.dto.response.History
 import com.sopt.kakaobank.databinding.FragmentHistoryBinding
+import java.text.DecimalFormat
 
 class HistoryFragment : BindingFragment<FragmentHistoryBinding>(R.layout.fragment_history) {
     override fun initView() {
-        // 로직 작성
+        statusBarColorOf(R.color.main) //status 색상 변경
+        setBackButton()
+        setHistoryList()
     }
+
+    // 뒤로 가기
+    private fun setBackButton() {
+        binding.btnHistoryBack.setOnClickListener {
+            findNavController().navigateUp()
+        }
+    }
+
+    // 1000원 단위 콤마
+    private fun formatAmount(amount : Int): String {
+        val dec = DecimalFormat("#,###")
+        return dec.format(amount)
+    }
+
+    private fun setHistoryList() {
```

Comment:
```suggestion
    private fun initHistoryAdapter() {
```
init[NameView]Adapter : 리사이클러뷰 어댑터 설정